tsconfig.build.json

```diff
@@ -14,7 +14,9 @@
     "sourceMap": true,
     "lib": ["es2018", "dom"],
     "moduleResolution": "node",
```

Comment:
might be worth mentioning this kinda changes in a way soon-ish: https://www.typescriptlang.org/tsconfig#moduleResolution